troubadix/plugins/__init__.py

```diff
@@ -135,7 +136,11 @@
     CheckVTPlacement,
     CheckWrongSetGetKBCalls,
     CheckOverlongDescriptionLines,
+<<<<<<< HEAD
     CheckSpacesInFilename,
+=======
+    CheckMultipleReParameters,
+>>>>>>> d1c48c5 (Add: Add plugin for multiple re parameters in script_mandatory_keys)
```

Comment:
Broken merge?

Reply:
fixed in c49dabeea6bd492e6bb3c3445bc14bd65a96d4cb